test/unit/protip/wrapper_test.rb

```diff
@@ -346,6 +371,13 @@ module Protip::WrapperTest # namespace for internal constants
     end
 
     describe '#get' do
+      before do
+        resource_class.class_exec(converter, inner_message_class) do |converter, message|
```

Comment:
It's defining a resource with no actions, a message of inner_message_class, and using the converter defined above